web/src/views/InvestigationDetail.vue

```diff
@@ -488,8 +489,14 @@ export default defineComponent({
     });
 
     const selectedPins: Ref<Pin[]> = computed(() => store.state.selectedPins);
+    /**
+     * Create note cards for any pins without a child dataset.
+     */
     function createPinNotes() {
-      pinNotes.value = store.state.currentPins.map((pin) => ({
+      const noteOnlyPins = store.state.currentPins.filter(
+        (pin: Pin) => (!pin.child && pin.note?.length),
+      );
```

Comment:
I thought we had discussed giving pins an explicit type field? That would make this type of filtering easier to understand in code, and easier to carry out at runtime.

Reply:
We have discussed this, and I'm not convinced we get all that much from such a field. I mostly think it would be redundant, since `pin.child.dataset_type` already exists. I could be talked into creating this field, but I don't see a need for it now.

Reply:
To summarize recent discussions: I think perhaps we are both wrong. We discussed on 2022-05-25 using Django more explicitly to model different types of pins, and I asked Mike to experiment with this idea on a branch based on this one. So, for the moment, this conversation can be resolved.